package.json

```
@@ -96,6 +96,7 @@
     "@typescript-eslint/eslint-plugin": "6.4.0",
     "@typescript-eslint/parser": "6.4.0",
     "@ungap/structured-clone": "1.2.0",
+    "@vscode/debugprotocol": "^1.61.0",
```

```suggestion
    "@vscode/debugprotocol": "1.61.0",
```

Just removing the `^` since we exclusively let Dependabot handle updates.